agent-model/src/test/java/com/newrelic/agent/model/LogEventTest.java

```diff
@@ -1,7 +1,47 @@
 package com.newrelic.agent.model;
 
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
 import static org.junit.Assert.*;
 
 public class LogEventTest {
-    // TODO
+
+    @Test
+    public void testConstructor() {
+        LogEvent logEvent = new LogEvent(Collections.emptyMap(), 0);
+
+        assertEquals(logEvent.getPriority(), 0, 0);
+        assertNotNull(logEvent.getMutableUserAttributes());
+        assertTrue(logEvent.getMutableUserAttributes().isEmpty());
+    }
+
+    @Test
+    public void testPriorityAccessors() {
+        LogEvent logEvent = new LogEvent(Collections.emptyMap(), 0);
+
+        assertEquals(0, logEvent.getPriority(), 0);
+
+        logEvent.setPriority(1);
+
+        assertEquals(1, logEvent.getPriority(),0);
+    }
+
+    @Test
+    public void testJsonString() throws IOException {
```

Comment:
Should `priority` be included in JSON string?

Reply:
Nope